### PR TITLE
#475 Fix save draft on browser

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/170965071-568b5a51-cee5-4b28-80eb-e3080e811b26.mov

https://user-images.githubusercontent.com/80730648/170965183-b1981e12-a4b0-4cd1-b378-7cbe856f4aa1.mov

### CODE REVIEW

**Comment by chibenwa** [comment]
What was the issue?

**Comment by dab246** [comment]
> What was the issue?

Due to the asynchronous mechanism, composer is deleted before it can save draft

**Comment by chibenwa** [comment]
Thanks for the explanation!

**Review by chibenwa — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

File: lib/features/composer/presentation/composer_controller.dart [hunk 1]
```diff
@@ -708,14 +719,22 @@ class ComposerController extends BaseController {
   }
 
   Future<bool> _isEmailChanged(BuildContext context, ComposerArguments arguments) async {
```
[the remark this answers is not in the document]
Comment (dab246) [reply lib/features/composer/presentation/composer_controller.dart:721]: 1. We currently do not support setting default values for identity.
2. And if yes default value is set. In your case the latest identity will be replaced in composer.

**Pull request merged**